src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -813,4 +726,22 @@ private boolean isUntrimmed(String value) {
     public static boolean isValidEmailAddress(String email) {
         return StringHelper.isMatching(email, REGEX_EMAIL);
     }
+
+    /////////////////////////////////////
+    // HELPER METHODS FOR INTERNAL USE //////////////////////////
+    // DUPLICATED IN FieldValidatorTest FOR EXTERNAL TESTS USE //
+    /////////////////////////////////////////////////////////////
```

Comment:
In fact, just make the following two methods public. As a utility class, this class can be called by anyone, but a test case should not be dependent on another, non-base test case, i.e `test.cases.*` -> `test.cases.*` is a dependency that should be avoided.